app/controllers/users_controller.rb

```diff
@@ -53,6 +53,17 @@ def update
     render :show, status: :internal_server_error
   end
 
+  def show_remove_user
+    @user = as_team_member(cognito_user: get_user(user_id: params[:user_id]))
+  end
+
+  def remove_user
+    @user = as_team_member(cognito_user: get_user(user_id: params[:user_id]))
+    UserDeletedEvent.create(data: { username: @user.email })
+    delete_user(username: @user.email)
```

Comment:
we should do some error handling here... show error message and don't fire `UserDeletedEvent`